FlashCap.Core/Internal/NativeMethods_AVFoundation.cs

```diff
@@ -402,12 +405,16 @@ protected NSObject(IntPtr handle, bool retain)
 
             protected override void Dispose(bool disposing)
             {
+                if (!disposing) return;
+                
                 if (Handle == IntPtr.Zero)
                     return;
 
-                LibObjC.SendNoResult(
+                LibCoreFoundation.CFRelease(Handle);
+                    
+                /*LibObjC.SendNoResult(
```

Comment:
This should be removed if not needed. If you are keeping it because you have some misgivings about it, please note in your comment what those misgivings are.